packages/components/pie-divider/src/divider.scss

```diff
@@ -1,54 +1,58 @@
-@use '@justeattakeaway/pie-css/scss' as p;
+@use '@justeattakeaway/pie-css/scss'as p;
```

Comment:
was this autoformatted? i don't see the suggestion in my local

Reply:
Think it's a typo tbh – I'll update